src/commands/functions/functions-create.js

```diff
@@ -189,6 +203,50 @@ const pickTemplate = async function ({ language: languageFromFlag }) {
 
 const DEFAULT_PRIORITY = 999
 
+const selectTypeOfFunc = async () => {
+  const functionTypes = [
+    { name: 'Edge function (Deno)', value: 'edge' },
+    { name: 'Serverless Function (Node/Go)', value: 'serverless' },
```

Comment:
The comment about not capitalising _function_ applied to serverless functions too.

```suggestion
    { name: 'Serverless function (Node/Go)', value: 'serverless' },
```